tests/integration/test_reads.py

```diff
@@ -753,6 +753,7 @@ def test_configure_row_group_batch_size(session_catalog: Catalog) -> None:
     assert len(batches) == entries
 
 
+@pytest.mark.integration
```

Comment:
Does CI fail if we don't add this line? It's what we're testing for in #1051

Reply:
It fails indeed. If I don't add that marker the `test-coverage-unit` command will try to run that test without initializing any docker container